<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleNotificationEditor</class>
 <widget class="QWidget" name="SingleNotificationEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="m_gbNotification">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sound</string>
        </property>
        <property name="buddy">
         <cstring>m_txtSound</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="m_txtSound">
          <property name="placeholderText">
           <string>Full path to your WAV sound file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnBrowseSound">
          <property name="text">
           <string>&amp;Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnPlaySound">
          <property name="text">
           <string>&amp;Play</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="m_cbBalloon">
        <property name="text">
         <string>Balloon notification</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_cbBalloon</tabstop>
  <tabstop>m_txtSound</tabstop>
  <tabstop>m_btnBrowseSound</tabstop>
  <tabstop>m_btnPlaySound</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
